apps/challenges/views.py

```diff
@@ -1,4 +1,5 @@
 import csv
+from datetime import datetime
```

Comment:
Please follow the correct import order. See the codebase for reference. All imports starting with `from ` appear at the end of `import` imports